packages/js/tailwind.config.js

```diff
@@ -75,6 +75,19 @@ module.exports = {
         '3xl': ['var(--nv-font-size-3xl)', { lineHeight: 'var(--nv-line-height-3xl)' }],
         '4xl': ['var(--nv-font-size-4xl)', { lineHeight: 'var(--nv-line-height-4xl)' }],
       },
+      backgroundImage: {
+        'dev-stripes-gradient':
+          'repeating-linear-gradient(135deg, rgba(255,132,71,0.07) 25%, rgba(255,132,71,0.07) 50%, rgba(255,132,71,0.11) 50%, rgba(255,132,71,0.11) 75%)',
+      },
+      animation: {
+        stripes: 'stripes 1s linear infinite',
+      },
+      keyframes: {
+        stripes: {
+          '0%': { transform: 'translateX(0)' },
+          '100%': { transform: 'translateX(calc(var(--stripes-size) * -1))' },
+        },
```

Comment:
the stripes animation is shown when hovering over the footer; maybe we should run it all time?